postman/AskDarcel%20Admin%20API.postman_collection.json

```diff
@@ -1,9 +1,10 @@
 {
+	"variables": [],
 	"info": {
 		"name": "AskDarcel Admin API",
-		"_postman_id": "a19cf529-de95-3738-10f3-2568497f9084",
+		"_postman_id": "241f6b3d-c648-de5c-61a0-6d4c5a7bfe58",
 		"description": "",
-		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json"
```

Comment:
Can you export the v2.1.0 schema version of the collection? I think that's why the diff is so large.